Source/Lib/Codec/EbPacketizationProcess.c

```diff
@@ -612,20 +606,21 @@ void* PacketizationKernel(void *inputPtr)
                 &sequenceControlSetPtr->bufferingPeriod,
                 sequenceControlSetPtr->videoUsabilityInfoPtr,
                 sequenceControlSetPtr->encodeContextPtr);
+
+            // Flush the Bitstream
+            FlushBitstream(
+                pictureControlSetPtr->bitstreamPtr->outputBitstreamPtr);
+
+            // Copy Buffering Period SEI to the Output Bitstream
+            CopyRbspBitstreamToPayload(
+                pictureControlSetPtr->bitstreamPtr,
+                &outputStreamPtr->pBuffer,
+                (EB_U32*) &(outputStreamPtr->nFilledLen),
+                (EB_U32*) &(outputStreamPtr->nAllocLen),
+                encodeContextPtr,
+                NAL_UNIT_INVALID);
```

Comment:
How about separating the PK logic correction into another PR or commit? Because it's different with the OUT_ALLOC related codes.

Reply:
will do